components/common/site_name_and_description.jsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+export default class SiteNameAndDescription extends React.PureComponent {
+    static propTypes = {
+
+        /**
```

Comment:
I know we discussed this at a developer meeting a while back, but do these comments add any value? I note they also duplicate in text the behaviour captured immediately below by `defaultProps`.